[2023-05-02 11:41:51,143] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.sensor_s3_obj manual__2023-05-02T11:41:45.968792+00:00 [queued]>
[2023-05-02 11:41:51,191] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.sensor_s3_obj manual__2023-05-02T11:41:45.968792+00:00 [queued]>
[2023-05-02 11:41:51,193] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-05-02 11:41:51,194] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2023-05-02 11:41:51,194] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-05-02 11:41:51,275] {taskinstance.py:1377} INFO - Executing <Task(S3KeySensor): sensor_s3_obj> on 2023-05-02 11:41:45.968792+00:00
[2023-05-02 11:41:51,283] {standard_task_runner.py:52} INFO - Started process 620 to run task
[2023-05-02 11:41:51,291] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_etl', 'sensor_s3_obj', 'manual__2023-05-02T11:41:45.968792+00:00', '--job-id', '487', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_taskflow.py', '--cfg-path', '/tmp/tmppe1b2bfd', '--error-file', '/tmp/tmpccsl93u4']
[2023-05-02 11:41:51,297] {standard_task_runner.py:80} INFO - Job 487: Subtask sensor_s3_obj
[2023-05-02 11:41:51,651] {task_command.py:370} INFO - Running <TaskInstance: dag_etl.sensor_s3_obj manual__2023-05-02T11:41:45.968792+00:00 [running]> on host 295d97741f96
[2023-05-02 11:41:52,220] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Victor
AIRFLOW_CTX_DAG_ID=dag_etl
AIRFLOW_CTX_TASK_ID=sensor_s3_obj
AIRFLOW_CTX_EXECUTION_DATE=2023-05-02T11:41:45.968792+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-02T11:41:45.968792+00:00
[2023-05-02 11:41:52,224] {s3.py:104} INFO - Poking for key : s3://raw/supermarket_sales.csv
[2023-05-02 11:41:52,302] {base.py:68} INFO - Using connection ID 'minio_conn' for task execution.
[2023-05-02 11:41:52,306] {base_aws.py:206} INFO - Credentials retrieved from login
[2023-05-02 11:41:57,662] {s3.py:104} INFO - Poking for key : s3://raw/supermarket_sales.csv
[2023-05-02 11:42:02,680] {s3.py:104} INFO - Poking for key : s3://raw/supermarket_sales.csv
[2023-05-02 11:42:07,707] {s3.py:104} INFO - Poking for key : s3://raw/supermarket_sales.csv
[2023-05-02 11:42:12,729] {s3.py:104} INFO - Poking for key : s3://raw/supermarket_sales.csv
[2023-05-02 11:42:17,750] {s3.py:104} INFO - Poking for key : s3://raw/supermarket_sales.csv
[2023-05-02 11:42:22,799] {s3.py:104} INFO - Poking for key : s3://raw/supermarket_sales.csv
[2023-05-02 11:42:22,832] {base.py:301} INFO - Success criteria met. Exiting.
[2023-05-02 11:42:22,966] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_etl, task_id=sensor_s3_obj, execution_date=20230502T114145, start_date=20230502T114151, end_date=20230502T114222
[2023-05-02 11:42:23,327] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-02 11:42:23,723] {local_task_job.py:273} INFO - 2 downstream tasks scheduled from follow-on schedule check
